experimental/segmenter/tests/complex_word.rs

```diff
@@ -38,6 +38,7 @@ fn word_break_th() {
     );
 }
 
+#[ignore]
```

Comment:
Nit: Can we document why this test is failed, e.g. `#[ignore = "please describe the reason"]`